content/en/blogs/install-kubernetes-containerd.md

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Install Kubernetes 1.22 and containerd the Easy Way'
+tag: 'Kubernetes, containerd'
+keywords: 'Kubernetes, containerd, docker, installation'
+description: 'Install Kubernetes and containerd in a Linux machine within minutes.'
+createTime: '2021-09-29'
+author: 'Feynman'
+snapshot: '/images/blogs/en/kubekey-containerd/k8s-containerd.png'
+---
+
+![k8s-containerd](/images/blogs/en/kubekey-containerd/k8s-containerd.png)
+
+[KubeKey](https://github.com/kubesphere/kubekey) is a lightweight and turn-key installer that supports the installation of Kubernetes, KubeSphere and related add-ons. Writtent in Go, KubeKey enables you to set up a Kubernetes cluster within minutes. In this blog, we will install Kubernetes 1.22 and [containerd](https://containerd.io/) in one command with KubeKey.
+
+## Step 1: Prepare a Linux Machine
+
+To get started with all-in-one installation, you only need to prepare one host according to the following requirements for hardware and operating system.
+
+### Hardware Recommendations
+
```

Comment:
@pixiake what are the minimum hardware requirements If we only install Kubernetes?

Reply:
Hi @FeynmanZhou 
You can refer to this [doc](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/#before-you-begin)

Reply:
Tks, updated.